content/docs/command-reference/import-url.md

```diff
@@ -55,16 +55,16 @@ source.
 
 DVC supports several types of (local or) remote locations (protocols):
 
-| Type     | Description                                         | `url` format                               |
-| -------- | --------------------------------------------------- | ------------------------------------------ |
-| `local`  | Local path                                          | `/path/to/local/data`                      |
-| `s3`     | Amazon S3                                           | `s3://mybucket/data`                       |
-| `azure`  | Microsoft Azure Blob Storage                        | `azure://my-container-name/path/to/data`   |
-| `gs`     | Google Cloud Storage                                | `gs://mybucket/data`                       |
-| `ssh`    | SSH server                                          | `ssh://user@example.com:/path/to/data`     |
-| `hdfs`   | HDFS to file (explanation below)                    | `hdfs://user@example.com/path/to/data.csv` |
-| `http`   | HTTP to file with _strong ETag_ (explanation below) | `https://example.com/path/to/data.csv`     |
-| `remote` | Remote path (see explanation below)                 | `remote://myremote/path/to/data`           |
+| Type     | Description                                         | `url` format                                 |
+| -------- | --------------------------------------------------- | -------------------------------------------- |
+| `local`  | Local path                                          | `/path/to/local/data`                        |
+| `s3`     | Amazon S3                                           | `s3://mybucket/data`                         |
+| `azure`  | Microsoft Azure Blob Storage                        | `azure://my-container-name/path/to/data`     |
+| `gs`     | Google Cloud Storage                                | `gs://mybucket/data`                         |
+| `ssh`    | SSH server                                          | `ssh://user@example.com/path/from/sftp/root` |
```

Comment:
I'm worried a bit about this exposing too much information that can just scary people when it's not needed.

e.g. regular SSH server on ubuntu works perfectly - I don't care if it's sftp or not. sftp in the path might confuse.

Reply:
True. I guess it's not something we need to bring up in these refs. Rolled back in 6ec6717f.